Vue2LeafletVectorGridProtobuf.vue

```diff
@@ -41,7 +27,9 @@ export default {
   },
   mounted () {
     this.mapObject = L.vectorGrid.protobuf(this.url, this.options)
-
+    if (this.options && this.options.interactive) {
+      L.DomEvent.on(this.mapObject, this.$listeners)
+    }
```

Comment:
This was already merged into master I think. Please rebase your branch to master.

Reply:
Yes, I did it. Thank you.